lib/SmartyPlugins/LMSSmartyPlugins.php

```diff
@@ -71,6 +73,7 @@ public static function buttonFunction(array $params, $template)
             . ' class="lms-ui-button' . ($type == 'link-button' ? ' lms-ui-link-button ' : '')
             . ($class ? ' ' . $class : '') . '"'
             . ($id ? ' id="' . $id . '"' : '') . ((($type == 'button' && empty($href)) || $type != 'button') && $onclick ? ' onclick="' . $onclick . '"' : '')
+            . ($target ? ' target="' . $target . '"' : '')
             . ($form ? ' form="' . $form . '"' : '')
```

Comment:
$target już miałeś jako parametr`external`, ale i tak mocno zmieniłem tę funkcję, żeby obsługiwała dowolne atrybuty podane jako parametry `{button}`. Istotne jest to, że znaki `_` z nazw parametrów zastępowane są znakami `-` w nazwach wynikowych atrybutów.
Commit: fe5aaba54

Reply:
Jeśli parametr nie ma `_` w nazwie np. `autofocus` to też zostanie odtworzony jako atrybut dla <button w kodzie HTML?

Reply:
> Jeśli parametr nie ma `_` w nazwie np. `autofocus` to też zostanie odtworzony jako atrybut dla <button w kodzie HTML?

Nie ma warunkowania odtwarzania atrybutów od występowania jakiegoś znaku w nazwie parametrów. Zresztą nie wynika to również z tego, co napisałem.

Reply:
Wolałem doprecyzować. Dzięki za info. Poszła poprawka jeśli chodzi o target.